### DISCUSSION TITLE
Welcome to owr-care-ha Discussions!

### DISCUSSION THREAD

**Discussion by zomco** [zomco/owcare-integration#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LiangyuLu-lly** [comment]
hi im lukas
im tring to conect my esp32s3 devtic -1 to the r60abd1 
but i dont know the progress
is that needs to flash the meshtastic to my esp?
but i cant find the proper device name
there is only somethings like nano rp and so on
so what should i do?
the fanial goal is i want to connect the r60abd1 to ha in the rp4b
i have watch your picture ,that is what i want
i tried a lot of ways ,but there still a lot of problems
like:the custom is removed,and i cant find me sensor or zhe lambe function is always something wrong
i hope you can help me cos its really important to me
By the way ,i saw your the sample picture is Chinese
so,are you a Chines?
cos i am .and we can type Chinese to conect
                                                              thank you

**Reply by zomco** [reply]
Sorry for my slow reply.
This repository has always been used by individuals. I have been too busy so I haven't maintained it for a long time. This repository needs to be used with the firmware developed by myself. It currently only supports esp32c3 (although the code supports esp32s3, it has not been fully tested).
If I understand correctly, your requirement is to read data from r60abd1 via UART on RP4B and then transfer the data to HA. To achieve the goal, you need to develop firmware running on RP4B and develop integration running on HA. Sorry, this repository is not suitable for your situation.
I can speak Chinese.

**Reply by LiangyuLu-lly** [reply]
感谢回复！我的ha是集成在树莓派4b上的，r60abd1则是连接在esp32s3上，我希望可以使esp32s3通过内网连接到ha，并且将它连接r60abd1的数据传到ha上
目前我有尝试过mqtt通讯，但是进度很慢，要是我买一个esp32c3可以用您的代码吗？如果可以的话，希望您告诉我具体的型号让我在网上购买一个
我觉得您代码已经写的十分完善，全GitHub上关于r60abd1最完善的，没有之一，这样停止有点可惜呢
希望收到您的回复

**Reply by zomco** [reply]
esp32c3版本仅适用于我自制的硬件，不适用于一般的esp32c3开发板。
因为esp32c3硬件资源有限，esp32s3与之相比有更好的扩展性，所以后续都会以esp32s3为主做迭代更新。计划未来一两周会先完成一个esp32s3开发板可用的版本，如果到时仍然感兴趣，可以体验一下。
感谢关注

**Reply by LiangyuLu-lly** [reply]
那真是太感谢了！！！这对我真的很重要！！！麻烦您要是方便的话，希望您将步骤写的稍微详细一点，对于初学者来说您的自述文件可能还是有点难以理解！感谢您！~（您可以帮我看看我这个esp32s3符合您的要求吗？还有这个r60abd1的雷达）
![微信图片_20250401124405](https://github.com/user-attachments/assets/5fb6269a-5201-4cf8-8079-1cd536e5977d)
![微信图片_20250401124414](https://github.com/user-attachments/assets/89f59ef4-cb10-483b-a1bd-93c2501fc8b7)

**Reply by zomco** [reply]
可以用的

**Reply by LiangyuLu-lly** [reply]
谢谢您，期待您伟大的作品

**Comment by LiangyuLu-lly** [comment]
你好啊，请问您开发得怎么样了？是否遇到了什么困难？
